lib/r10k/git/shellgit.rb

```diff
@@ -53,6 +53,33 @@ def clean(opts = {})
         end
       end
 
+      def fetch(remote, opts = {})
+        cmd = ["fetch"]
```

Comment:
since there aren't any conditionals this might be easier to read if you just defined the array